persist/sqldb/workflow_archive.go

```diff
@@ -153,7 +153,7 @@ func (r *workflowArchive) ListWorkflows(namespace string, name string, minStarte
 		From(archiveTableName).
```

Comment:
this changes the func signature (it is not `name` anymore, it is `namePrefix`

Reply:
good catch